core/commands/commands_test.go

```diff
@@ -120,18 +120,7 @@ func TestCommands(t *testing.T) {
 		"/name/pubsub/subs",
 		"/name/resolve",
 		"/object",
-		"/object/data",
 		"/object/diff",
-		"/object/get",
-		"/object/links",
-		"/object/new",
-		"/object/patch",
-		"/object/patch/add-link",
-		"/object/patch/append-data",
-		"/object/patch/rm-link",
-		"/object/patch/set-data",
```

Comment:
I feel we should keep `patch add-link` and `patch rm-link`. 
These are useful for patching new files into existing directories.

They won't have a real alternative until https://github.com/ipfs/kubo/issues/4782 is implemented. 

I don't think MFS covers all the bases, people need something solid that is thread-safe, and we know MFS as it is implemented right now (a single global DAG),  runs into issues when multiple threads start mutating it. 

By keeping `add-link` and `rm-link`  we provide people with an alternative to MFS until they can have more than one global MFS root.